yarn.lock

```diff
@@ -1576,6 +1576,11 @@
     schema-utils "^3.0.0"
     source-map "^0.7.3"
 
+"@remix-run/router@1.4.0":
```

Comment:
did you install yarn? for what reason? if you already had npm installed it is not a good idea to install yarn and use both